style.css

```diff
@@ -0,0 +1,147 @@
+:root {
+  --color-main_theme: #36b37e;
+  --color-second_green: #c5e9da;
+  --color-light-green: #ebf0ee;
+  --color-dark-blue: #172b4d;
+  --color-white-bg: #e5e5e5;
+  --color-white: #fff;
+  --color-button-enabled: #36b37f;
+  --color-button-pressed: #008552;
+  --color-button-hover: rgba(54, 179, 127, 0.24);
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+  font-size: 1.6rem;
+}
+
+.header {
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+header {
+  width: 100vw;
+  max-width: 800px;
+  height: 5vh;
+  min-height: 65px;
+  background-color: var(--color-white-bg);
+}
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+}
+
+nav ul {
+  list-style: none;
+  display: none;
+}
+
+.burger {
+  display: flex;
+  width: 18px;
+  height: 12px;
+  margin-top: 25px;
+  margin-right: 14px;
+  flex-direction: column;
+  align-self: flex-start;
+  justify-content: space-around;
+  cursor: pointer;
+}
```

Comment:
- [ ] I think it would be better to add a bit more space/padding/margin to the right of the hamburger menu to get a design close to the guidelines as much as possible 📌 What do you think? 💭

Your design | Design guidelines
---| ---|
![image](https://user-images.githubusercontent.com/106119105/234728494-0fb3fd67-92a2-4158-9aa8-9240cd582929.png) | ![image](https://user-images.githubusercontent.com/106119105/234728544-fb9912fe-e325-476a-98bc-645d082a0d49.png)